### DISCUSSION TITLE
모바일 버전에서 리더보드 SetScore 적용안되는 이슈

### DISCUSSION THREAD

**Discussion by lhj5709** [naverz/zepeto-studio-kor#514] [Issue Report (Legacy)]
유니티 에디터 상에서는 작동하지만 모바일 앱으로 빌드해서 테스트하니 아무런 반응이 없습니다,,

데이터를 가져오는 것은 동작하는데 SetScore부분만 안되네요.  문제가 있는 걸까요 ?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 관련 내용 내부 확인 중에 있습니다.
조금만 기다려 주세요~ 
감사합니다.
